deployment/gce/lily/docker_init.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -exo pipefail
+
+mkdir -p ${LILY_REPO}/keystore
+
+
+if [[ ! -z "${LILY_DOCKER_INIT_IMPORT_MAINNET_SNAPSHOT}" ]]; then
+  # set default snapshot path if not already defined
+  snapshot="${LILY_DOCKER_INIT_IMPORT_SNAPSHOT_PATH:-https://snapshots.mainnet.filops.net/minimal/latest}"
```

Comment:
If `LILY_DOCKER_INIT_IMPORT_SNAPSHOT_PATH` is nil, then the snapshot is a URL. You'll need to `GET` the resource either with `curl` or `aria2c` and then untar the zst compressed snapshot.

Reply:
For the command description: https://github.com/filecoin-project/lily/blob/master/commands/daemon.go#L96. It can read the url.

I had test it before, it works!

Reply:
Awesome!